src/sss_client/pam_sss_prompt_config.c

```diff
@@ -45,8 +45,9 @@ struct prompt_config_passkey {
     char *prompt_touch;
 };
 
-struct prompt_config_sc_pin {
-    char *prompt; /* Currently not used */
+struct prompt_config_smartcard {
+    char *prompt_init;
```

Comment:
`prompt_inst` or `prompt_instruction`  to match more closely to what is mention in the design page?